frontends/web/src/common/Annotation/CreateInterface.js

```diff
@@ -879,7 +879,7 @@ class CreateInterface extends React.Component {
                             </OverlayTrigger>
                           )}
 
-                          {this.state.task.cur_round > 1 && (
+                          {/* {this.state.task.cur_round > 1 && (
```

Comment:
also, I think it might be worth including a label or gui element stating the current Round that the user is submitting for even if they can't change the round

Reply:
Sure that would be nice!

Reply:
![image](https://user-images.githubusercontent.com/53152779/144552647-b246bffe-2b55-4c92-83c8-25571179b349.png)

This is how the label looks as per my latest [commit](https://github.com/facebookresearch/dynabench/pull/834/commits/34b79fe555f69dc1bda6d8c6240491c8a6ad1634)